controllers/webhooks/label_indexer/webhook.go

```diff
@@ -34,7 +34,6 @@ func NewWebhook() *LabelIndexerWebhook {
 				LabelRule{Label: korifiv1alpha1.CFRouteHostLabelKey, IndexingFunc: Unquote(JSONValue("$.spec.host"))},
 				LabelRule{Label: korifiv1alpha1.CFRoutePathLabelKey, IndexingFunc: SHA224(Unquote(JSONValue("$.spec.path")))},
 				LabelRule{Label: korifiv1alpha1.CFRouteIsUnmappedLabelKey, IndexingFunc: IsEmptyValue(JSONValue("$.spec.destinations[*]"))},
-				MultiLabelRule{LabelRules: DestinationAppGuidLabelRules},
```

Comment:
With this change, the `MultiLabelRule` becomes unused, therefore you could also delete [this code](https://github.com/cloudfoundry/korifi/blob/db631edd57d41251833ea57af946ba84fdee182c/controllers/webhooks/label_indexer/rules/labels.go#L29-L51) alongside its tests